go.mod

```diff
@@ -13,46 +13,53 @@ require (
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
 	github.com/santhosh-tekuri/jsonschema/v5 v5.0.0
-	github.com/spf13/cobra v1.5.0
+	github.com/spf13/cobra v1.6.1
 	github.com/spf13/viper v1.7.1
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.8.1
 	github.com/xeipuuv/gojsonschema v1.2.0
-	
+
 	// 24.1.23: A new version of kubeconform will be released soon, using a different json-schema library.
 	// This affects our output when resources fail schema validation. Do not update kubeconform version
 	// without consulting the product team
 	github.com/yannh/kubeconform v0.4.14
-	
 	gopkg.in/op/go-logging.v1 v1.0.0-20160211212156-b2cb9fa56473
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	k8s.io/apimachinery v0.23.5
 	sigs.k8s.io/yaml v1.2.0
 )
 
+require github.com/open-policy-agent/opa v0.49.2
+
 require (
+	github.com/OneOfOne/xxhash v1.2.8 // indirect
```

Comment:
are those extra packages are necessary? `xxhash`

Reply:
I run `go mod tidy` and it didn't change